docs/documentation/release_notes/topics/25_0_0.adoc

```diff
@@ -42,6 +42,10 @@ per-hashing request.
 To prevent excessive memory and CPU usage, the parallel computation of hashes by Argon2 is by default limited to the number of cores available to the JVM.
 To support the memory intensive nature of Argon2, we have updated the default GC from ParallelGC to G1GC for a better heap utilization.
 
+Note that Argon2 is not compliant with FIPS 140-2. So if you are in the FIPS environment, the default algorithm will be still PBKDF2. Also note that if you are on non-FIPS environment and
+you plan to migrate later to the FIPS environment, it can be good idea to explicitly switch password policy to some FIPS compliant algorithm like `pbkdf2-sha512` from the beginning as otherwise
```

Comment:
```suggestion
you plan to migrate later to the FIPS environment, it can be good idea to explicitly switch the password policy to some FIPS compliant algorithm like `pbkdf2-sha512` from the beginning as otherwise
```